_posts/2021-06-010-How-to-stop-getting-notifications.md

```diff
@@ -0,0 +1,158 @@
+---
+date: 2021-06-10
+title: How to stop getting notifications
+video_id: JC2yJnBXzLM
+description: These steps are about How to stop getting notification of leads or how to disable them.
+type:  
+sidebar:
+  - {id: step-1, text: Stop all notification }
+  - {id: step-2, text: my assistant only}
+  - {id: step-3, text: Click notification}
+  - {id: step-4, text: Disable receive email notification}
+  - {id: step-5, text: Specific notification}
+  - {id: step-6, text: Instant notification}
+  - {id: step-7, text: Weekly summer notification}
+  - {id: step-8, text: Long term schedule}
+  - {id: step-9, text: EmailCC notification}
+  - {id: step-10, text: Clicking link notification feature}
+  - {id: step-11, text: How to open notification from gmail}
+  - {id: step-12, text: Notification preferences}
+  
+
+categories:
+  - How-To
+# set: getting-started
+# set_order: 3
+---
+
+Sometimes you're getting continous email notifications or you want to edit your preferences.
+
+To change notifications setting or disable any notification, follow the steps mentioned below
+
+
+
+
+## Step-1 
+
+How to stop getting all notifications.
+
+Go to "assistant", if you are Org admin you'll see all the assistants.
+
+![image](../../images/Stop-getting-notification-1.png)
```

Comment:
The image is not correct. You made a red square on Leads, while the clicked menu is "Assistant". See the underline below Assistant